src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -434,7 +434,7 @@
     ;; amq send thread
     (future
       (try
-        (let [s (.createSession connection true Session/SESSION_TRANSACTED)
+        (let [s (.createSession connection true 0)
```

Comment:
I'm not quite across why we can't use the const symbols here, and are instead using the numbers. Session/SESSION_TRANSACTED translates to an integer of 0 doesn't it?